hunt/app/views/animals/show.html.erb

```diff
@@ -1,2 +1,9 @@
-<h1>Animals#show</h1>
-<p>Find me in app/views/animals/show.html.erb</p>
+
+<%= link_to '<button type="button" class="btn btn-dark">View all Animals</button>'.html_safe, '/' %>
```

Comment:
I didnt mention it before, but we can apply classes using the `link_to` helper so that we dont have to use `html_safe` here. Rails also gives us some path helpers like you used on the index page. If you run `rake routes` there is a column called "Prefix" and you can add `_path` to the end of that prefix for use here. So, in all, this could be changed to:

`<%= link_to 'View All Animals', animals_path, class: 'btn btn-dark' %>`